lib/Twig/Extension/Core.php

```diff
@@ -1463,15 +1463,25 @@ function twig_get_attribute(Twig_Environment $env, Twig_Source $source, $object,
         throw new Twig_Error_Runtime('Accessing Twig_Template attributes is forbidden.');
     }
 
+    static $extensionSandbox = array();
```

Comment:
I've put up a different implementation in #2361 . Have also added what would appear to be a missing call to update the options hash. In terms of the non-static class property - the code I was originally modifying wasn't within a class..